src/PageTournament/LoadTournament.res

```diff
@@ -41,18 +41,23 @@ let isLoadedDone = x =>
   }
 
 @react.component
-let make = (~children, ~tourneyId, ~windowDispatch=_ => ()) => {
+let make = (~children, ~tourneyId, ~windowDispatch=?) => {
```

Comment:
This component and the mock for it were actively fighting against me, I have no idea why no combination I've tried works

Reply:
This confused me for a while too, but I think I figured it out. The behavior for `include module type of` seems to interact poorly with the new version's behavior of `@react.component`. I think the simplest fix right now is to just remove the .resi file. Its only purpose was to be an extra way to check that the mock adequately matched the original component.

Reply:
Thanks for your tip, I deleted the .resi and also did some more updates and replaced the deprecated `ReactDOMStyle.make`. At least locally the webapp started up for the first time :tada: - but the whole build system based on webpack is still really out-of-date but I don't really understand the current setup to migrate it to something more modern and simple like `vite` so I'll leave that untouched for now.

Reply:
Yeah, the build system is indeed quite out-of-date. It's made from vendored create-react-app code that I heavily modified by mostly removing the features I didn't need. I'm sure that the whole thing could eventually be replaced with something more modern.